app/views/accounts.scala.html

```diff
@@ -1,33 +1,49 @@
-@import com.gu.janus.model.{AwsAccount, AccountOwners, JanusData}
+@import com.gu.janus.model.{AwsAccount, JanusData}
 @import com.gu.googleauth.UserIdentity
 @import play.api.Mode
+@import scala.util.Try
 
-@(accountOwners: List[(AwsAccount, AccountOwners)], user: UserIdentity, janusData: JanusData)(implicit mode: Mode, assetsFinder: AssetsFinder)
+@import com.gu.janus.model.Permission
+@import scala.util.Failure
+@import scala.util.Success
+@import logic.ViewHelpers
+@(accountOwners: List[(AwsAccount, List[(String, Set[Permission])], Try[String])], user: UserIdentity, janusData: JanusData)(implicit mode: Mode, assetsFinder: AssetsFinder)
 
 @main("Accounts", Some(user), janusData) {
     <div class="container" xmlns="http://www.w3.org/1999/html">
         <h1 class="header orange-text">Accounts</h1>
+        @janusData.permissionsRepo match {
+          case Some(permissionsRepo) => {
+              <p>
+                  Refer to <a href="@permissionsRepo">Janus' configuration</a> for full details on who has what access to each AWS account.
+              </p>
+          }
+          case None => {
+              <p>Refer to Janus' configuration for full details on who has what access to each AWS account.</p>
+          }
+        }
         <ul class="collapsible collapsible-accordion">
-        @for((account, owners) <- accountOwners) {
+        @for((account, owners, tryAccountNumber) <- accountOwners) {
```

Comment:
I suppose these aren't really account 'owner's
```suggestion
        @for((account, users, tryAccountNumber) <- accountOwners) {
```

Reply:
The old thinking behind the name "owners" here, is that people with access to an account are people that are respinsible for the account. In some sense, everyone that appears in this list is an "owner". I agree this becomes murkier now that the access levels are more tightly scoped in many cases, and I agree that the word owners isn't totally accurate here anymore. I decided to defer changing this for now, and I'll revisit it after we've done a bit of tidying, renaming and consolidating of the existing account access levels.

Great catch, I should have mentioned this in the PR description.